src/application/tasks/ventilation.c

```diff
@@ -65,6 +65,7 @@ void ventilation(){
 
 void start_inspiration(){
   uint32_t current_time = get_current_time();
+  uint32_t breath_cycle_duration = calculate_breath_cycle_duration(); //Added this line to calculate breath cycle (not sure if compatible with int variable)

```

Comment:
good 

Although when you want to discuss something this is a great way to include your message make sure to delete this comment for the final version of the code

Reply:
Gotcha, I'll be changing this to a constant since this is what is breaking the first test (previously it was a value instead of a function call.

Reply:
Gotcha, deleted comment since the function should be there anyway.